fixtures/ASPNetTemplateApplication/Scripts/README.md

```diff
@@ -0,0 +1,219 @@
+<!-- IGNORE THE HTML BLOCK BELOW, THE INTERESTING PART IS AFTER IT -->
```

Comment:
Can this fixture be simplified? Not sure how we feel about checking in this js/css if it's not necessary to test compression.

Reply:
We could add this extra functionality to the existing fixture but we are unable to build the checked in fixture app in order to add it. We now think that the best option going forward would be for us to maintain a published `nora` that can be built from VS and submodule the published fixture everywhere .NET apps are used across cloudfoundry. Are y'all open to this idea?